### DISCUSSION TITLE
Midi2LR update

### DISCUSSION THREAD

**Discussion by robreyez** [rsjaffe/MIDI2LR#750] [General]
Any news on a new MIDI2LR update...or is all officially lost?. hehe alot of new features getting added to LR.

**Comment by RBCAMERA** [comment]
I for one would be glad to pay for an update. This is an incredibly useful tool and I see no reason the developer should not be rewarded, and given an incentive to keep updating.

**Comment by hdstead** [comment]
I would also contribute some $$$.

**Comment by 153957** [comment]
Note; there is a `Sponsor` button for this repository. And of course we just have to wait and see if @rsjaffe feels up to it to update / continue work on this project ( https://github.com/rsjaffe/MIDI2LR/issues/746#issuecomment-1732340773 )

**Comment by hdstead** [comment]
Ok, thanks for the update.

On Sun, Oct 22, 2023 at 2:13 PM Arne de Laat ***@***.***>
wrote:

> Note; there is a Sponsor button for this repository. And of course we
> just have to wait and see if @rsjaffe <https://github.com/rsjaffe> feels
> up to it to update / continue work on this project ( #746 (comment)
> <https://github.com/rsjaffe/MIDI2LR/issues/746#issuecomment-1732340773> )
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rsjaffe/MIDI2LR/discussions/750#discussioncomment-7353070>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AL23NPCUVAM7VQDVUFS7I5TYAWEARAVCNFSM6AAAAAA6KGFRG6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TGNJTGA3TA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 















*​                                ''~`                             ( o o
)+---------------oooO--(_)--Oooo.-------------------------------+|| Doug
Stead,  Chairman:|Tri-M Group of Companies|www.Tri-M.Com
<http://www.Tri-M.Com>
 |                                                         |  "I believe in
the impossible because no one else does."|              - Florence Griffith
Joyner -|   |                     .oooO |                         (
)       Oooo.   +---------------------\ (---- (
)------------------------------------+                              \_)
) /                                     (_/*

**Comment by rsjaffe** [comment]
I'm trying to get restarted on this project after laying it aside for a while. Help with the MacOS app is particularly needed.

**Reply by 153957** [reply]
I a not very familiar in programming in Lua & C++, but I do have an Apple Silicon Mac (also an Intel one) and am willing to help. Perhaps try making builds, test, or debug issues.

**Comment by rsjaffe** [comment]
There is a test update out (see announcement at the top of the discussion group page). Once it's been tested a bit, I'll put it out for general release, assuming there's no serious issues.
